src/v/raft/vote_stm.cc

```diff
@@ -268,8 +275,8 @@ ss::future<> vote_stm::update_vote_state(ssx::semaphore_units u) {
           ec.value(),
           ec.message());
     } else {
-        vlog(_ctxlog.info, "became the leader term: {}", term);
         if (term == _ptr->_term) {
```

Comment:
nit: consider adding an else case that logs when we successfully replicated the config, but that the term has changed? I guess it may be obvious in the logs, since we'd likely see news of the term change in other messages.